1040.rb

```diff
@@ -0,0 +1,26 @@
+n1 = gets.to_f
+n2 = gets.to_f
+n3 = gets.to_f
+n4 = gets.to_f
+
+media = (2*n1 + 3*n2 + 4*n3 + 1*n4)/10
+mediar = media.round(1)
+puts "Media: #{mediar}"
+
+if mediar >= 7.0
+    puts "Aluno aprovado."
+elsif mediar < 5.0
+    puts "Aluno reprovado."
+else 
+    puts"Aluno em exame."
+    exame = gets.to_f
+    media_final = (mediar + exame)/2
+    media_finalr = media_final.round(1)
```

Comment:
```suggestion
    media_final_arredondada = media_final.round(1)
```

Seria interessante evitar abreviar muito os nomes, dessa forma você tem um código mais legível e que representa de fato o que você está fazendo.

Reply:
Boa, vou adotar isso